hack/dlv-tssc-mcp-server.sh

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
+#
+# This script wraps "dlv exec" on "tssc mcp-server", keeping the STDIO
+# communication and exposing Delve API on a local port.
+# 
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+# Current script directory.
+declare -r SCRIPT_DIR="$(dirname ${0})"
+
+# Installer executable, by default relative to this script.
+declare -r TSSC_BIN="${TSC_BIN:-${SCRIPT_DIR}/../bin/tssc}"
+# Installer container image, a required flag for the mcp-server subcommand.
+declare -r TSSC_IMAGE="${TSSC_IMAGE:-ghcr.io/redhat-appstudio/tssc:latest}"
```

Comment:
Why not give the option to use a local image? Look at the usage of `deploy.sh` (`--container`).